packages/docs/docs/tutorial/integrate-sdk/android/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Android",
+  "position": 2
```

Comment:
`"position": 1.1` for both android and react sdk is good. we need leave `1` to introduction

Reply:
updated